modules/curl/8.8.0.bcr.1/patches/add_build_file.patch

```diff
@@ -276,16 +276,16 @@ index 0000000..ff61c6f
 +        ],
 +        "@platforms//os:macos": [
 +            "-Wl,-framework,SystemConfiguration",
-+            "-lpthread",
++            "-Wl,-framework,CoreFoundation",
++            "-Wl,-framework,Security",
 +        ],
 +        "//conditions:default": [
 +            "-lpthread",
 +        ],
 +    }),
 +    local_defines = select({
 +        "@platforms//os:macos": [
-+            'CURL_CA_BUNDLE=\\"/etc/ssl/cert.pem\\"',
```

Comment:
I think we need to leave these, since they're required if mbedtls is used, I imagine that means we'll need a similar condition to the one you added for boringssl for this? seems like the framework linking is pretty harmless to have always but I assume these defines are mutually exclusive

Reply:
AFAICT, the macros are unused if Secure Transport is enabled:

https://github.com/curl/curl/blob/5637ca1aab33b861f7b2c903150cafe01b896c41/lib/url.c#L431-L437

So I left the macros as they were and minimized the linking because it was not difficult to do.

Reply:
Ping?

Reply:
i think what you have now is good, that this setting overrides that behavior, but this is the right default, thanks!